docs/Legal/Customer Agreement.mdx

```diff
@@ -0,0 +1,85 @@
+# Acuerdo de Servicio
+
+Al pagar, usar y aceptar el acuerdo de cualquier servicio ("Servicio") proporcionado por Harmony Ltd., usted acepta quedar obligado por los siguientes términos y condiciones ("Acuerdo"). Por favor, léalos detenidamente.
+
+## Términos y Condiciones
+
+### Aceptación de los Términos
+
+Este Acuerdo describe los términos bajo los cuales se proporciona el Servicio a usted ("Cliente"). Al acceder o utilizar el Servicio, usted acepta cumplir y quedar obligado por este Acuerdo.
+Si no está de acuerdo con estos términos, por favor no utilice el Servicio.
+
+### Descripción del Servicio
+
+Harmony proporciona una versión personalizada de nuestro sistema de gestión de ONG que permitirá a los usuarios gestionar beneficiarios, alimentos, atención y cualquier otro servicio proporcionado por la ONG a sus beneficiarios.
+
+### Uso del Servicio
+
+Usted acepta utilizar el Servicio únicamente para fines legales y de acuerdo con este Acuerdo. Usted es responsable de todas las actividades realizadas bajo su cuenta.
+
+### Cuentas de Usuario
+
+Las cuentas de usuario solo pueden ser creadas por nosotros o un miembro autorizado de la ONG. Usted acepta ser responsable de cualquier violación de la ley de protección de datos que pueda proporcionar acceso a cualquier persona fuera de su organización
+o cometerla usted mismo.
+
+### Política de Privacidad
+
+Su privacidad es importante para nosotros. Nuestra **Política de Privacidad** explica cómo recopilamos, usamos y protegemos su información personal, la cual se mantiene al mínimo y solo se utiliza con el propósito
+de mantener el Servicio funcionando según lo previsto.
+
+### Pago
+
+Si decide utilizar nuestros servicios, acepta pagar las tarifas asociadas con ello. Las tarifas se acuerdan entre ambas partes y pueden ser actualizadas de vez en cuando. Usted acepta pagar las tarifas de manera oportuna
+y proporcionarnos un método de pago válido. Si su método de pago falla repetidamente, nos reservamos el derecho de suspender o cancelar su acceso al Servicio, aunque primero se recopilará y enviará una copia de los datos almacenados
+a usted.
+
+### Propiedad Intelectual
+
+El nombre y el logotipo son propiedad intelectual de Harmony por derechos de autor. Cualquier uso indebido de estos recursos será perseguido.
+
+### Terminación
+
+Podemos terminar o suspender su acceso al Servicio de inmediato, con previo aviso y enviando una copia de los datos de sus beneficiarios resultantes de un incumplimiento de este Acuerdo.
+
+### Limitación de Responsabilidad
+
+Harmony no será responsable de ningún daño indirecto, incidental, especial, consecuencial o punitivo que resulte de su uso o incapacidad para usar el Servicio.
```

Comment:
I could mention the existence of a manual but there is no such thing at this point, if that is what you mean

Reply:
By manual I mean a video or instructions on how to use our system correctly, it is true that currently there is no such resource but it will be made available later.